scripts/generate_new_lineage_definitions.bash

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash 
+curl https://raw.githubusercontent.com/cov-lineages/pango-designation/refs/heads/master/pango_designation/alias_key.json > alias_key.json 
+curl https://raw.githubusercontent.com/cov-lineages/pango-designation/refs/heads/master/lineages.csv | tail -n +2 | cut -d',' -f2 > lineages.csv
```

Comment:
Why not simply download the whole file, save it as is and do the tedious parsing in Python instead of weird bash commands?

Reply:
What python file? This would necessitate another python script as the current transformation script has nothing to do with all the additional information contained in the downloaded file. So this would require this script to then call another python file.. I do not find that simpler than the invocation of 2 commands. What exactly would be your alternate suggestion?

Reply:
What does it do? It cuts off a few lines and it basically strips down the CSV to a single column.

IMO it's more understandable to do that in Python code. `alias2lineageDefinitions.py` reads the file anyway, so it can also do those two simple transformations.

But if you think it's easier, we can also leave it here.

Reply:
I did make a second script that does some generalized version of this :)